### DISCUSSION TITLE
Broken tooltip when creating repo

### DISCUSSION THREAD

**Discussion by SiddharthShyniben** [community/community#4813] [Other Feature Feedback, Questions, & Ideas] [answered]
Looks like the tooltip on the new repo page is broken.

![Screen Shot 2021-07-28 at 9 59 00 AM](https://user-images.githubusercontent.com/76640268/127263893-219c7a9e-7241-46f9-b53f-7a787aa3aeec.png)

It covers up the text

**Comment by davidstaheli** [comment, marked as answer]
@SiddharthShyniben, thank you for reporting this!  It should be fixed now.

![image](https://user-images.githubusercontent.com/1767415/131113423-f00f606f-253e-4363-b094-bab37a53cb7b.png)
